x-pack/plugins/security_solution/public/common/containers/use_full_screen/index.tsx

```diff
@@ -46,10 +45,10 @@ export const useGlobalFullScreen = (): GlobalFullScreen => {
   const setGlobalFullScreen = useCallback(
     (fullScreen: boolean) => {
       if (fullScreen) {
-        document.body.classList.add(SCROLLING_DISABLED_CLASS_NAME);
+        document.body.classList.add(SCROLLING_DISABLED_CLASS_NAME, 'euiDataGrid__restrictBody');
         resetScroll();
       } else {
-        document.body.classList.remove(SCROLLING_DISABLED_CLASS_NAME);
+        document.body.classList.remove(SCROLLING_DISABLED_CLASS_NAME, 'euiDataGrid__restrictBody');
         resetScroll();
       }
 
```

Comment:
are we able to dispatch setFullScreen for global full screen as well? I could find some style issue due to incorrect global full screen status?
<img width="1669" alt="Screenshot 2021-09-02 at 13 58 04" src="https://user-images.githubusercontent.com/6295984/131847783-b3dc29b7-c24a-4c9d-a8e5-d333b230146e.png">

Reply:
ya this is an unrelated z-index issue that was/will be fixed by upgrading eui, https://github.com/elastic/kibana/pull/109926 via https://github.com/elastic/eui/pull/5054 I think

Reply:
https://github.com/elastic/kibana/pull/110913/files/7a964244aea4b4c1c899fc1e405b06aa4db6b941#diff-9dafa871a9206508107bdcf810febcdd5e7689609dc09fcca94ade9210ce49aeR55 inputActions.setFullScreen is being dispatched in both callbacks as well

Reply:
Update: turns out it was not fixed, but fixed in this commit https://github.com/elastic/kibana/pull/110913/commits/0c1da5a20d0d322967af0cd9e407861680a08808